pkg/resource/promtail-wiring/reconciler.go

```diff
@@ -44,7 +44,7 @@ func (r *Reconciler) ReconcileCreate(ctx context.Context, lc loggedcluster.Inter
 	}
 
 	desiredApp := lc.WirePromtail(currentApp)
-	if !reflect.DeepEqual(currentApp, desiredApp) {
+	if !reflect.DeepEqual(currentApp, *desiredApp) {
```

Comment:
:exploding_head: 
How could it ever work before? It could never update the wiring? :astonished: